tests/preprocessing/test_filter.py

```diff
@@ -24,11 +44,13 @@ def test_filter_staypoints(self):
         
         assert len(within_spts) == gis_within_num, "The spatial filtered sp number should be the same as" + \
             "the one from the result with ArcGIS"
-        assert all(within_spts.geometry == intersects_spts.geometry), "For sp the result of within and" + \
-            "intersects should be the same"
         assert len(crosses_spts) == 0, "There will be no point crossing area"
         
+        # For staypoints the result of within and intersects should be the same
```

Comment:
@hong2223 within the comments, what is the meaning of _within_ and _intersects_ here?

Reply:
These correspond to the `method` parameter in `spatial_filter` [function](https://github.com/mie-lab/trackintel/blob/master/trackintel/preprocessing/filter.py). These correspond to the shapely provided so-called _binary predictors_, see for example [here ](https://shapely.readthedocs.io/en/stable/manual.html#object.intersects)